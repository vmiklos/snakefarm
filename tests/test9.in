debug 1
out 1
time 100
load test9.map
addplayer 1
addsnake 1 1 0 1 1 0
18 0
17 0
endsnake
addplayer 1
addsnake 2 2 3 1 1 0
19 0
12 0
endsnake
comment # az elso jatekos kigyojanak meg kell halnia, a masodiknak nem
step
exit