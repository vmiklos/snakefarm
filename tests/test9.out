Event MapLoad test9.map
# az elso jatekos kigyojanak meg kell halnia, a masodiknak nem
Event Step 1
StepEvent Snake_Snake 2
StepEvent SnakeDie
End Step
Event Step 2
End Step
Event Exit